documentation/user_docs/docs/manual/Horace_and_Libisis_installation.rst

```diff
@@ -103,7 +103,7 @@ Supporting package (Herbert or Libisis) will be the package, initiated last. If
 Installation using package_on file
```

Comment:
Is Libisis a thing?

Reply:
Not since about 2008... It should probably be removed... I haven't really read through all the docs...

Reply:
Added a warning that this page is out of date. Will probably remove it in a future update.